src/core/sock/sockinfo.h

```diff
@@ -644,4 +649,80 @@ class sockinfo : public socket_fd_api,
     //////////////////////////////////////////////////////////////////
 };
 
+typedef std::unordered_map<flow_tuple, bool> port_no_bind_map_t;
+struct port_map_list {
+    std::list<in_port_t>::iterator it;
+    port_no_bind_map_t map;
```

Comment:
This type is used only once. I'd suggest to remove typedef and put std::unordered_map here. This will simplify reading the code without harm to writing.

Reply:
done.